ng/src/web/pages/portlists/dialog.js

```diff
@@ -52,11 +52,12 @@ const PortListsDialog = ({
   name = _('Unnamed'),
   port_list,
   port_range = 'T:1-5,7,9,U:1-3,5,7,9',
+  tmp_port_ranges = [],
```

Comment:
Could you rename tmp_port_ranges to only port_ranges? The dialog doesn't know if the port_ranges are temporary  :smiley:

Reply:
The dialog doesn't know that, but then again djafodnic would be an equally senseful variablename for the dialog ;) I was aiming at letting the reader know, what this variable is used for and which larger context stands behind it.